src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -49,14 +50,29 @@ public PersonCard(Person person, int displayedIndex) {
         this.person = person;
         id.setText(displayedIndex + ". ");
         name.setText(person.getName().fullName);
-        phone.setText(person.getPhone().value);
         address.setText(person.getAddress().value);
         email.setText(person.getEmail().value);
+        priority.setText(person.getPriority().toString());
+        setPriorityColor(person.getPriority().value);
         person.getTags().stream()
                 .sorted(Comparator.comparing(tag -> tag.tagName))
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
     }
 
+    private void setPriorityColor(Priority.Level level) {
+        String priorityColor = "";
+        if (level == Priority.Level.LOW) {
```

Comment:
Perhaps you can consider using a switch statement? Just a suggestion.

Reply:
Yup switch statement will be better. Will change.